src/main/java/com/example/onlinebookshop/services/impl/UserDetailsServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package com.example.onlinebookshop.services.impl;
```

Comment:
```suggestion
package com.example.onlinebookshop.security;
```
replace to another package